iacrcc/iacrcc.cls

```diff
@@ -182,15 +195,19 @@
       pdfvolumenum={\IACR@vol},%
       pdfissuenum={\IACR@no},%
       pdfpagerange={\IACR@fp-\IACR@lp},%
+      pdfnumpages=\IACR@lp,%
```

Comment:
I am doubtful that `\IACR@lp` is set correctly in the preamble, but I think this is nothing new. Something to be fixed in the future.

Reply:
To confirm this, if you compile test14 and then look at the main.pdf file, you see 
```
<prism:pageRange>1-??</prism:pageRange>
```
which suggests that `\IACR@lp` is not being set correctly. I think we should file an independent issue on this but submit this change now.

Reply:
Interesting, I have been inspecting the pdf meta information with exiftool and fixed the problem in 10feba5ce1f478930b65542b27f4c6ded198b177.